defaults.py

```diff
@@ -1,4 +1,6 @@
 import datetime
+from typing import List, Dict, Tuple, Union, Optional, Any, Callable, Iterable, Set, TypeVar, Generic, Sequence

```

Comment:
Is there a reason to import all these? not just the ones we use or will probably use?

Reply:
Don't know what we will "probably" use, so just imported a bunch of ones. Not sure if this is a bad practice or not

Reply:
Fine, maybe it's just me with my OCD of seeing such a long import. But I see many more in the module so * will be heavier. FINE